keel-artifact/src/main/kotlin/com/netflix/spinnaker/keel/artifacts/DebianArtifactSupplier.kt

```diff
@@ -36,14 +36,6 @@ class DebianArtifactSupplier(
   override val supportedVersioningStrategy =
     SupportedVersioningStrategy("deb", DebianVersioningStrategy::class.java)
 
-  override fun publishArtifact(artifact: PublishedArtifact) {
```

Comment:
we don't need this (it's duplicated to the `shouldProcess` logic below).

Reply:
This is pretty surprising to me, given the description of the PR. This is the function that gets called by `ArtifactController` to publish a new artifact. Has the base class been modified to call `shouldProcessArtifact`?

Reply:
`artifactListener` is calling this. It was in previous PRs.